### PR TITLE
[IMP] im_livechat: remove event 'close_chat_window' in public livechat code

### PR DESCRIPTION
**Pull request by alexkuhn**
Task-2964389

### CODE REVIEW

File: addons/im_livechat/static/src/legacy/widgets/public_livechat_window/public_livechat_window.js [hunk 1]
```diff
@@ -74,7 +74,17 @@ const PublicLivechatWindow = Widget.extend({
      * @override
      */
     close() {
-        this.trigger_up('close_chat_window');
+        const isComposerDisabled = this.messaging.publicLivechatGlobal.chatWindow.widget.$('.o_thread_composer input').prop('disabled');
```
Comment (oomsveta) [inline addons/im_livechat/static/src/legacy/widgets/public_livechat_window/public_livechat_window.js:77]:
```suggestion
        const isComposerDisabled = this.widget.$('.o_thread_composer input').prop('disabled');
```

File: addons/im_livechat/static/src/legacy/widgets/public_livechat_window/public_livechat_window.js [hunk 2]
```diff
@@ -74,7 +74,17 @@ const PublicLivechatWindow = Widget.extend({
...
+        const shouldAskFeedback = !isComposerDisabled && this.messaging.publicLivechatGlobal.messages.find(function (message) {
+            return message.id !== '_welcome';
+        });
+        if (shouldAskFeedback) {
+            this.messaging.publicLivechatGlobal.chatWindow.widget.toggleFold(false);
```
Comment (oomsveta) [inline addons/im_livechat/static/src/legacy/widgets/public_livechat_window/public_livechat_window.js:82]:
```suggestion
            this.widget.toggleFold(false);
```

File: addons/im_livechat/static/src/legacy/widgets/public_livechat_window/public_livechat_window.js (hunk 1 above)
Comment (alexkuhn) [reply addons/im_livechat/static/src/legacy/widgets/public_livechat_window/public_livechat_window.js]: change done

File: addons/im_livechat/static/src/legacy/widgets/public_livechat_window/public_livechat_window.js (hunk 2 above)
Comment (alexkuhn) [reply addons/im_livechat/static/src/legacy/widgets/public_livechat_window/public_livechat_window.js]: change done

Comment (alexkuhn) [reply addons/im_livechat/static/src/legacy/widgets/public_livechat_window/public_livechat_window.js:82]: done

**Review by oomsveta — APPROVED** [review]

**Comment by alexkuhn** [comment]
Merged in grouped PR: https://github.com/odoo/odoo/pull/99137

**Pull request closed without merging**